### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by mrparkersson**
Pull request
Amo Parker Prince

What does this PR do?

Designing React bookstore template in the component directory

Description of the task to be completed
A tests using Jest and React test library

Used React in writing JSX and adding Redux store to handle states later in the template

created test.MD files that have been ignored
How should it be tested
By running npm test in the terminal.

clone the repository
git clone https://github.com/mrparkersson/React-Bookstore
navigate to the folder
cd bookstore
checkout the feature

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,60 @@
...
 
-In the project directory, you can run:
+- React
 
-### `npm start`
+## Online live link
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+[Visit project online](https://github.com/mrparkersson/React-Bookstore)
```
Comment (mikethreels) [inline README.md]: - [x] the Live demo is not set up, please create a working link to the live demo or remove this part of the readme file entirely.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,60 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+This project is [MIT](./MIT.md) licensed.
```
Comment (mikethreels) [inline README.md]: - [x] the link to the license is not working as intended, please have a look at [the template](https://github.com/microverseinc/readme-template) provided by Microverse to see how it is supposed to work.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Parker, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
### Because the below requirements haven't been changed in this PR I'm not able to leave an inline comment, therefore I'll leave a comment in this message.
- [x] according to the requirements you should add React Redux this can be done by running `npm install react-redux`
- [x] according to the requirements you should structure your application files using a "feature folder" approach and use the [ducks pattern](https://github.com/erikras/ducks-modular-redux) for your Redux files. however, there is no Redux folder at all.
- [x] there are no components setup yet, please have a look at the below requirements:
```
Add React Router and set two <Route>s and <Link>s for the app's navigation:

    Books - the default view
        Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet)
        Should have a form for adding a book (no functionality yet)
    Categories
        Should display "Under construction" text only.
```

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (mrparkersson) [reply README.md]: please checkout feature branch

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @mrparkersson,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a Code Review Request that has no linters set up

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Parker, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**